.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish documentation
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think we shouldn't release the site on master, as the documentation depends on current code and we don't want it to reflect unreleased version.

That said, docusaurus does support versioned documentation but I don't think the plugin supports that